### DISCUSSION TITLE
Serializer/deserializer for all enums that changes names of enums values

### DISCUSSION THREAD

**Discussion by hajdamak** [FasterXML/jackson-databind#4705] [Q&A]
Hi,

what would the best way to implement custom enum deserializer/serializer that changes names of enum values?
It should work for all enums.

Regards,

**Comment by JooHyukKim** [comment]
There should be plenty documentation/resources on web.

**Comment by cowtowncoder** [comment]
That is a very vague question. Changes names (... or values?) how? Please include an example of what you mean (it sounds like you have some concrete idea).
